synquacer.mk

```diff
@@ -0,0 +1,127 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+COMPILE_NS_USER   ?= 64
+COMPILE_NS_KERNEL ?= 64
+COMPILE_S_USER    ?= 64
+COMPILE_S_KERNEL  ?= 64
+
+DEBUG=0
```

Comment:
s/`DEBUG=0`/`DEBUG ?= 0`/ for consistency ?

Reply:
sounds good.

Reply:
edit pushed.